lib/msf/core/post/file.rb

```diff
@@ -578,6 +579,32 @@ def copy_file(src_file, dst_file)
 
 protected
 
+  def _read_file_powershell(filename)
+    eof = cmd_exec("[System.IO.File]::ReadAllBytes(\"#{filename}\").Length").strip.to_i
+    data = ''
+    start = 0
+    chunk_size = 1024 * 1024
+    loop do
+      stop = start + chunk_size
+      stop = eof if stop > eof
+      data << _read_file_powershell_fragement(start, stop, filename)
+      start += chunk_size
+      break if start >= eof
+    end
+    return data
+  end
+
+  def _read_file_powershell_fragement(start, stop, filename)
```

Comment:
I know this is a private method, but since it uses the same prefix as `_read_file_powershell` and the arguments it takes is a superset of those, it would be nice if the common arguments were kept in the same order:

```suggestion
  def _read_file_powershell_fragment(filename, chunk_size, offset=0)
```
Note that the suggestion also fixes the spelling of "fragment" and uses the arguments from the pseudo code suggestion.